nvflare/app_common/statistics/numeric_stats.py

```diff
@@ -160,14 +156,18 @@ def accumulate_hists(
             if feature not in global_hists:
```

Comment:
this line although not changed in this PR.

Should this be `if feature not in global_hists[ds_name]`?